src/components/Avatar.tsx

```diff
@@ -87,15 +88,20 @@ function Avatar({
     const isWorkspace = type === CONST.ICON_TYPE_WORKSPACE;
     const userAccountID = isWorkspace ? undefined : (avatarID as number);
 
-    const source = isWorkspace ? originalSource : getAvatar({avatarSource: originalSource, accountID: userAccountID});
+    const source = isWorkspace ? originalSource : getAvatar({avatarSource: originalSource, accountID: userAccountID, defaultAvatars});
     let optimizedSource = source;
     const maybeDefaultAvatarName = getPresetAvatarNameFromURL(source);
 
     if (maybeDefaultAvatarName) {
         optimizedSource = getAvatarLocal(maybeDefaultAvatarName);
     }
-    const useFallBackAvatar = imageError || !source || source === Expensicons.FallbackAvatar;
-    const fallbackAvatar = isWorkspace ? getDefaultWorkspaceAvatar(name) : fallbackIcon || Expensicons.FallbackAvatar;
+    const useFallBackAvatar = imageError || !source || source === defaultAvatars.FallbackAvatar;
+    let fallbackAvatar;
+    if (isWorkspace) {
+        fallbackAvatar = getDefaultWorkspaceAvatar(name);
+    } else {
+        fallbackAvatar = fallbackIcon === undefined ? defaultAvatars.FallbackAvatar : fallbackIcon;
```

Comment:
I think we can make it shorter, wdyt? Same for all ternary operators of condition `xxxx === undefined`
```suggestion
        fallbackAvatar = fallbackIcon ?? defaultAvatars.FallbackAvatar;
```